stdlib/public/stubs/CMakeLists.txt

```diff
@@ -1,33 +1,40 @@
-set(swift_stubs_objc_sources)
-set(swift_stubs_unicode_normalization_sources)
+set(swift_stubs_sources
```

Comment:
On Darwin we don't use ICU for String functions (hashing, comparison) but call to Foundation's NSString/CFString functions. On non-darwin platforms we call to ICU for this functionality.
The stubs for calling to ICU are in UnicodeNormalization.cpp.